.github/workflows/build-wheel-linux-arm64.yaml

```diff
@@ -14,12 +14,29 @@ on:
     # Thursdays we test the standalone plugin
     - cron: '35 4 * * 4'
   workflow_dispatch:
+    inputs:
+      build_standalone_plugin:
+        description: 'Build the standalone plugin wheel'
+        required: false
+        default: 'false'
   workflow_call:
+    inputs:
+      build_standalone_plugin:
+        description: 'Build the standalone plugin wheel'
+        type: boolean
+        required: false
+        default: false
 
 concurrency:
   group: Build Catalyst Wheel on Linux (arm64)-${{ github.ref }}
   cancel-in-progress: true
 
+env:
+  # If the `inputs.build_standalone_plugin` is set to a value (true/false), then that is used.
+  # If the input is empty (meaning this workflow was not triggered by a workflow_call/workflow_dispatch), then check if event_name is schedule.
+  # If event_name is NOT schedule, default to false
+  BUILD_STANDALONE_PLUGIN: ${{ inputs.build_standalone_plugin || github.event_name == 'schedule' || 'false' }}
+
```

Comment:
I'm confused by this variable, isn't it the same as before and still entirely dependent on whether the event trigger is a schedule?

Reply:
Oh wait, having a value regardless of true or false uses that value in a github or (`||`) statement?

Reply:
> I'm confused by this variable, isn't it the same as before and still entirely dependent on whether the event trigger is a schedule?


It let's us move away from having to rely on the *value* of the schedule, which is what we currently rely on to determine if the standalone plugin should be built or not

Reply:
> Oh wait, having a value regardless of true or false uses that value in a github or (||) statement?

The possible values for the input is `true`/`false`/`undefined (or NULL)`, the third value is when this workflow is not called from workflow_dispatch or workflow_call, so then the github or `||` goes to the next value

Reply:
Ah I see, we can account enforce a type a little better, no problem!

Reply:
I'm still confused, can you confirm if this is true in github or not?
> what I'm wondering is if `false || ...` just evaluates to false

and whether that is what this comment is implying?
> If the `inputs.build_standalone_plugin` is set to a value (true/false), then that is used.

Reply:
Yeah if the value is false (as a boolean), the OR would indeed go to the next expression. I have updated the logic so any value for the input is casted to string, where any value would be used as it is truth-y.

Apologies for the mishap there

Reply:
@dime10 I (think) the issue is resolved now.

This is now the behavior as I have tested:
- `true` and `'true'` gets casted to string value `'true'`
- `false` and `'false'` gets casted to string value `'false'`
- The input being missing gets casted to string value `''`

The first two values are truth-y and the third value is false-y.

This yields the expected result stated in PR description and in the comment.

Let me know if you have further concerns.

Reply:
Nice I think that looks pretty sensible! :) 

Would you mind explaining what the `|| false` does in that line?

Reply:
Done!

Reply:
@dime10 good to mark this as resolved?